bin/drop_counts.py

```diff
@@ -4,134 +4,128 @@
 import re
 from collections import OrderedDict
 from pathlib import Path
-
+import sys
 import pandas as pd
```

Comment:
I would make this less verbose and only import the functions you actually need :)
```suggestion
from pandas import DataFrame, read_csv, etc...
```

Reply:
Done